src/Pie.js

```diff
@@ -57,37 +91,42 @@ const Pie = ({ sections, radius, innerRadius, backgroundColor,strokeCap}) => {
           stroke={backgroundColor}
           strokeWidth={width}
         />
-        <RingShape radius={radius} width={width} color={backgroundColor} />
+        <ArcShape radius={radius} width={width} color={backgroundColor} startAngle={0} arcAngle={360} />
         {sections.map((section, idx) => {
           const { percentage, color } = section;
 
-          const startAngle = startValue / 100 * 360
-          const arcAngle = percentage / 100 * 360
-          startValue += percentage
-
-          return arcAngle >= 360
-            ? <RingShape
-                key={idx}
-                radius={radius}
-                width={width}
-                color={color}
-              />
-            : <ArcShape
-                key={idx}
-                radius={radius}
-                width={width}
-                color={color}
-                startAngle={startAngle}
-                arcAngle={arcAngle}
-                strokeCap={strokeCap}
-              />
+          const startAngle = startValue / 100 * 360;
+          const arcAngle = getArcAngle(percentage);
+          startValue += percentage;
+
+          shouldShowRoundDividers && paintedSections.push({ percentage, color, startAngle, arcAngle });
+
+
+          return <ArcShape
+            key={idx}
+            radius={radius}
+            width={width}
+            color={color}
+            startAngle={startAngle + dividerSize}
```

Comment:
`startAngle={startAngle + dividerSize/2}`?

Reply:
but I think it should be `startAngle={startAngle + dividerSize/2}` and `endAngle={endAngle - dividerSize/2}`

Reply:
@zgordon01 can you make sure this and I'm ready for a new release

Reply:
Now I get your point here. Took me a bit of thought why to divide by 2. Because otherwise we would be taking away from total % of the pie

Reply:
nope, for example, if we have [20, 30, 40] which is [0-20, 20-50, 50-90] and divider size to 2, it will be [0-1, 1-19, 19-21, 21-49, 49-51, 51-89, 89-90]